### PR TITLE
nginx ubuntu and centos 7

### PR DESCRIPTION
**Pull request by jburel**
install nginx version from distribution

Check that version 1.10 is installed (travis)

### CODE REVIEW

**Comment by joshmoore** [comment]
```
# ubuntu
Get:17 http://ppa.launchpad.net/nginx/stable/ubuntu xenial/main amd64 nginx all 1.10.3-0+xenial0 [24.2 kB]

# centos
--> Processing Dependency: nginx-filesystem = 1:1.10.2-1.el7 for package: 1:nginx-1.10.2-1.el7.x86_64
```
etc.

File: linux/install_ubuntu1604_nginx.sh [hunk 1]
```diff
@@ -34,7 +34,7 @@ su - omero -c "OMEROVER=$OMEROVER ICEVER=$ICEVER bash -eux step04_all_omero.sh"
 
 su - omero -c "bash setup_omero_db.sh"
 
-OMEROVER=$OMEROVER bash -eux step05_ubuntu1404_nginx.sh
+OMEROVER=$OMEROVER bash -eux step05_ubuntu1406_nginx.sh
```
Comment (joshmoore) [inline linux/install_ubuntu1604_nginx.sh]: `1604` no? Looks like this passed without failure leaving something ppa installed on 1604 which might point to a deeper issue.

Comment (jburel) [reply linux/install_ubuntu1604_nginx.sh]: Let me fix that and check

**Comment by jburel** [comment]
Fixed see ``Setting up nginx (1.10.3-0+xenial0) `` (travis job output)

**Comment by joshmoore** [comment]
Did you happen to notice how/why the ppa for nginx was installed even when the `1406` script didn't exist?

**Comment by jburel** [comment]
Found the problem. The wrong script has been used in docker file.
Retesting it

**Comment by jburel** [comment]
This is better ``Setting up nginx (1.10.0-0ubuntu0.16.04.4)``
Pushing a fix shortly

**Comment by jburel** [comment]
I have found another small version mismatch now that the correct install script is used.
I can fix in this PR or in a follow up PR. It will require c77140a to be tested

(default on 16.04 is pg 9.5)

**Comment by jburel** [comment]
pg issue fixed in https://github.com/ome/omero-install/pull/135

**Comment by joshmoore** [comment]
```
$ git grep -E "1[46]0[46]_(nginx|pg)"
.travis.yml:  - ENV=ubuntu1404_nginx
.travis.yml:  - ENV=ubuntu1604_nginx
linux/README.md:    bash install_ubuntu1404_nginx.sh
linux/README.md:    bash install_ubuntu1604_nginx.sh
linux/install_ubuntu1404_nginx.sh:bash -eux step01_ubuntu1404_pg_deps.sh
linux/install_ubuntu1404_nginx.sh:OMEROVER=$OMEROVER bash -eux step05_ubuntu1404_nginx.sh
linux/install_ubuntu1604_nginx.sh:bash -eux step01_ubuntu1604_pg_deps.sh
linux/install_ubuntu1604_nginx.sh:OMEROVER=$OMEROVER bash -eux step05_ubuntu1604_nginx.sh
linux/test/README.md:    ./docker-build.sh ubuntu1404_nginx
linux/test/README.md:    docker run --rm -it -p 8080:80 -p 4063:4063 -p 4064:4064 omero_install_test_ubuntu1404_nginx
linux/test/README.md:JAVAVER=oracle18 ./docker-build.sh ubuntu1404_nginx
linux/test/README.md:    PGVER=pg95 ./docker-build.sh ubuntu1404_nginx
linux/test/README.md:    docker run --rm -it -p 8080:80 -p 4063:4063 -p 4064:4064 omero_install_test_ubuntu1404_nginx
linux/test/README.md:`step01_ubuntu1404_pg_deps.sh`, `step01_ubuntu1604_pg_deps.sh` and update this README.md.
linux/test/README.md:    ICEVER=ice35-devel ./docker-build.sh ubuntu1404_nginx
linux/test/README.md:    WEBSESSION=true ./docker-build.sh ubuntu1404_nginx
linux/test/README.md:    docker run --rm -it -p 8080:80 -p 4063:4063 -p 4064:4064 omero_install_test_ubuntu1404_nginx --websession true
linux/test/README.md:    WEBAPPS=true ./docker-build.sh ubuntu1404_nginx
linux/test/README.md:    docker run --rm -it -p 8080:80 -p 4063:4063 -p 4064:4064 omero_install_test_ubuntu1404_nginx
linux/test/README.md:    OMEROVER=OMERO-DEV-latest ./docker-build.sh ubuntu1404_nginx
linux/test/ubuntu1404_nginx/Dockerfile: bash install_ubuntu1404_nginx.sh && \
linux/test/ubuntu1604_nginx/Dockerfile: bash install_ubuntu1604_nginx.sh && \
```

Certainly looks reasonable. Grep'ing just for `1[46]0[46]` is slightly confusing since several `1404` scripts are used for `1604`.

**Comment by joshmoore** [comment]
```
Get:2 http://ppa.launchpad.net/nginx/stable/ubuntu/ trusty/main nginx-common all 1.10.3-0+trusty0 [45.5 kB]

vs. 

Get:5 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 nginx-common all 1.10.0-0ubuntu0.16.04.4 [26.6 kB]
```

:+1:

**Comment by jburel** [comment]
I did not want to duplicate the script.
I could rename name them to "ubuntu" only in another PR

**Comment by joshmoore** [comment]
Might be something to consider in a follow-up. (Especially if the file names appear anywhere in the documentation)

**Pull request merged**